### PR TITLE
Fix parameter type for onOnLockscreenNotification

### PR DESCRIPTION
**Pull request by adein**
Change method parameter type for onOnLockscreenNotification from (SDLLockscreenStatus *) to (SDLOnLockscreenStatus *).

### CODE REVIEW

**Comment by joeljfischer** [comment]
I'm going to reformat before pushing 4.0.0 final, but it should be noted that this would change a lot of formatting from correct to incorrect. This ought to only change the one line in order to keep the git history clean.

**Comment by adein** [comment]
Yeah, I don't know why all of those changes are in here.

File: SmartDeviceLink-iOS/SmartDeviceLink/SDLProxyListener.h [hunk 1]
```diff
@@ -57,69 +57,69 @@
 @class SDLUnregisterAppInterfaceResponse;
 @class SDLUnsubscribeButtonResponse;
 @class SDLUnsubscribeVehicleDataResponse;
+@class SDLOnLockscreenStatus;
```
Comment (joeljfischer) [inline SmartDeviceLink-iOS/SmartDeviceLink/SDLProxyListener.h]: This should be placed in correct alphabetical order.

**Comment by adein** [comment]
@joeljfischer Same question: will these be fixes during the merge, or should Infosys fix them in their PR branch?

**Comment by joeljfischer** [comment]
Hmm...ideally it should be done before the merge, so that it is it's own separate commit, and not a part of the merge commit.

**Comment by joeljfischer** [comment]
:+1: LGTM

**Pull request merged**